apps/browser-extension/src/css/sprinkless.css.ts

```diff
@@ -0,0 +1,145 @@
+import { createSprinkles, defineProperties } from '@vanilla-extract/sprinkles'
+import { colors, spacing, borderRadius } from '@reown/appkit-ui-new'
+
+const flexAlignments = ['flex-start', 'flex-end', 'center']
+
+const dimensions = {
+  '1': '1px',
+  '2': '2px',
+  '4': '4px',
+  '8': '8px',
+  '12': '12px',
+  '20': '20px',
+  '24': '24px',
+  '28': '28px',
+  '30': '30px',
+  '32': '32px',
+  '34': '34px',
+  '36': '36px',
+  '40': '40px',
+  '44': '44px',
+  '48': '48px',
+  '49': '49px',
+  '50': '50px',
+  '54': '54px',
+  '56': '56px',
+  '62': '62px',
```

Comment:
I've used colors, spacing and borde radius from the `@reown/appkit-ui-new` package but for dimensions i needed to expand it a bit more so will use that for now